twistr/headline.go

```diff
@@ -0,0 +1,18 @@
+package twistr
+
+import "fmt"
+
+func Headline(s *State) {
+	usaHl := SelectCard(s, USA, TheChinaCard)
+	sovHl := SelectCard(s, SOV, TheChinaCard)
+	// XXX: If space race show headline is in play, deal with that yo
+	MessageBoth(s, fmt.Sprintf("USA selects %s, and SOV selects %s", usaHl.Name, sovHl.Name))
```

Comment:
we're referring to usa and sov players externally as "US" and "USSR", which is how the actual game does it